### PR TITLE
Fixed some typos and added Keep On the Grass

### PR DESCRIPTION
**Pull request by macaquedev**
I don't think this editorial is my best work, so @myst-6 can you please edit it to make it more understandable/readable? Thanks a lot.

### CODE REVIEW

**Comment by macaquedev** [comment]
oops... I accidentally was working in the wrong branch (again!!) and submitted my solution to R1 2001 Q3 into this branch instead of making a pull request.

**Comment by myst-6** [comment]
Please don't do this next time. I'll review them both at once this time, but if that happens again I'll ask you to revert the commit

File: solutions/bio2/KeepOnTheGrass/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      <STitle>Observation 2</STitle>
+      <SText>
+        The diagrams given in the problem statement are
+        deliberately misleading, because they do not quite
+        give enough squares to show how the pattern proceeds.
+        However, by generating the next few "terms" of the
+        pattern, we can see that the pattern is actually
+        periodic - if we let $f(x, y)$ be the colour of
+        the square at coordinates $(x, y)$, then we have
+        $f(x, y) = f(x + 4n, y + 4m)$ for all integers $n, m$.
```
Comment (myst-6) [inline solutions/bio2/KeepOnTheGrass/index.tsx:25]: diagram is wrong

**Review by myst-6 — CHANGES_REQUESTED** [review]

File: solutions/bio2/KeepOnTheGrass/index.tsx (hunk 1 above)
Comment (macaquedev) [reply solutions/bio2/KeepOnTheGrass/index.tsx:25]: ouch... I guess the diagram given in the official BIO2 editorial turned out to be the misleading one. LOL

**Review by myst-6 — APPROVED** [review]

**Review by vladimirfilip — APPROVED** [review]
LGTM!

**Pull request merged**